v2/pkg/genruntime/importable_resource.go

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package genruntime
+
+// ImportableResource is implemented by any resource that can be imported into the operator via asoctl
+type ImportableResource interface {
+	// InitializeSpec is used to set up the spec for a resource based on the STATUS that's currently available
```

Comment:
Crossing up Spec and Status like this is uncommon enough that I want to have an extra comment being explicit that it was deliberate. Otherwise some damn fool developer (that is, me in six months) is going to come along and thing the original author was a doofus and "fix" it.

Reply:
It just seems like a weird comment to put in "public" documentation, which this is since it's in `pkg`.

It'd be another thing if it were in `internal`?

Reply:
Removed.